src/services/activation/model.js

```diff
@@ -0,0 +1,59 @@
+const mongoose = require('mongoose');
+
+const { Schema } = mongoose;
+const { Configuration } = require('@schul-cloud/commons');
```

Comment:
Codacy found an issue: ['Configuration' is assigned a value but never used.](https://app.codacy.com/manual/schul-cloud/schulcloud-server/pullRequest?prid=5660869)